app/app/services/aggregators/response_objects/identity.rb

```diff
@@ -26,7 +26,7 @@ def self.from_argyle(identity_response_body)
         full_name: identity_response_body["full_name"],
         date_of_birth: identity_response_body["birth_date"],
         emails: [ identity_response_body["email"] ],
-        ssn: identity_response_body["ssn"],
+        ssn: (identity_response_body["ssn"]&.last(4) if identity_response_body["ssn"].present?),
```

Comment:
Thoughts on obfuscating it similar to how we do for pinwheel?

```suggestion
        ssn: ("XXX-XX-#{identity_response_body["ssn"]&.last(4)}" if identity_response_body["ssn"].present?),
```

I guess this doesn't handle the case where SSN is only two digits long. Not sure if we're likely to hit that or if Argyle will give us back a proper length SSN all the time.